README.md

```diff
@@ -544,6 +553,14 @@ __AWESOME__ apps counter: __183__ 🎉
 - [x] [GitHub](https://github.com/thundernest/k-9)
 - [x] [Official page](https://k9mail.app/)
 
+### SimpleLogin
```

Comment:
Is this app an Email client overall of just for SimpleLogin?

Reply:
Not an Email client, it helps in managing email aliases.

Reply:
Where should I add it then?

Reply:
Hmm... I think it can stay here however how about:
## Email
### FairEmail — Client
<img alt="FairEmail" height="64" src="https://raw.githubusercontent.com/M66B/FairEmail/master/app/src/main/res/mipmap-hdpi/ic_launcher.png">

- [x] [Google Play](https://play.google.com/store/apps/details?id=eu.faircode.email)
- [x] [F-Droid](https://f-droid.org/en/packages/eu.faircode.email/)
- [x] [Github](https://github.com/M66B/FairEmail)
- [x] [Official page](https://email.faircode.eu/)

### K-9 Mail — Client
<img alt="K-9Mail" height="64" src="https://f-droid.org/repo/com.fsck.k9/en-US/icon_-2bZW0ZnkKqPVher2SxQK8hXGGSjgaBHoa8x6vW0v8w=.png">

- [x] [Google Play](https://play.google.com/store/apps/details?id=com.fsck.k9)
- [x] [F-Droid](https://f-droid.org/packages/com.fsck.k9/)
- [x] [GitHub](https://github.com/thundernest/k-9)
- [x] [Official page](https://k9mail.app/)

### SimpleLogin — Aliases Management
<img alt="SimpleLogin" height="64" src="https://raw.githubusercontent.com/simple-login/Simple-Login-Android/master/SimpleLogin/app/src/main/ic_launcher-playstore.png">

- [x] [Google Play](https://play.google.com/store/apps/details?id=io.simplelogin.android)
- [x] [F-Droid](https://f-droid.org/en/packages/io.simplelogin.android.fdroid/)
- [x] [GitHub](https://github.com/simple-login/Simple-Login-Android)
- [x] [Official page](https://simplelogin.io/)

or something like that

Reply:
This look like a bit excessive 
Either we should add aliases management in bracket with simple login, or we should make a new category

Reply:
Yeah, however see [Trackers section](https://github.com/Psyhackological/AAA#trackers=).
I like this approach with "in bracket" style.

Reply:
Well, it looks good, but  what about using indentation in trackers section also

Reply:
### Original
## 2FA
### Aegis 
<img alt="Aegis" height="64" src="https://github.com/beemdevelopment/Aegis/blob/master/app/src/main/res/drawable/ic_aegis_iconx.png?raw=true">

- [x] [εxodus report](https://reports.exodus-privacy.eu.org/en/reports/com.beemdevelopment.aegis/latest/): 0 trackers, 3 permissions
- [x] [F-Droid Anti-Features](https://f-droid.org/en/packages/com.beemdevelopment.aegis/): 0
- [x] [Privacy Policy](https://getaegis.app/aegis/privacy.html): no bits collection.
> Aegis Authenticator does not collect any data from your device.

### Indentation
## 2FA
### Aegis 
<img alt="Aegis" height="64" src="https://github.com/beemdevelopment/Aegis/blob/master/app/src/main/res/drawable/ic_aegis_iconx.png?raw=true">

> [εxodus report](https://reports.exodus-privacy.eu.org/en/reports/com.beemdevelopment.aegis/latest/): 0 trackers, 3 permissions
[F-Droid Anti-Features](https://f-droid.org/en/packages/com.beemdevelopment.aegis/): [Privacy Policy](https://getaegis.app/aegis/privacy.html): no bits collection.
Aegis Authenticator does not collect any data from your device.

__I don't know. It looks worse to me.__